### PR TITLE
Add content

### PR DESCRIPTION
**Pull request by delisma**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+	layout: splashpage.html
```
Comment (nschonni) [inline index.html:2]: might need to remove the leading whitespace, but I'm not sure

File: index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	permalink: design-conception.canaca.ca/index.html
```
Comment (nschonni) [inline index.html]:
```suggestion
	permalink: design-conception.canada.ca/index.html
```

File: index.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+	description-fr: Le site Web du gouvernement du Canada fournit un point d'accès complet à tous les programmes, services, départements, ministères et organismes du gouvernement du Canada.
+	creator-en: Government of Canada, Service Canada, Citizen Service Branch, Integrated Channel Management, Web Strategies and Product Management
+	creator-fr: Gouvernement du Canada, Service Canada, Direction générale de service aux citoyens, Gestion intégrée des modes de service, Gestion des stratégies et produits Web
+	subject-en: Government of Canada, services
+	subject-fr: Gouvernement du Canada, services
+	lang: en
+	section: message
+	dateModified: 2019-12-19
+---
+<div class="sp-hb">
```
Comment (nschonni) [inline index.html:16]: It seems like most of this should be coming from the root template, and this page just should be metadate only for the template values

**Review by nschonni — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (Borges-Porteous) [reply index.html]: I think we should also just stick to design.canada.ca for the bilingual repo.

Both Montreal and the Quebec governments are using "design" not conception. In @quidampepin 's words, we don't want to be more catholic than the pope.

File: index.html (hunk 3 above)
Comment (Borges-Porteous) [reply index.html:16]: We've also been adding a JSON-LD snippet for schema.org metadata, contextual to the page. How can we keep that when using jekyll?

See example on [radio buttons page](https://github.com/canada-ca/design-system/blob/master/components/radio-buttons.html)

File: index.html (hunk 2 above)
Comment (delisma) [reply index.html]: @Borges-Porteous The design-conception.canada.ca is the URL for the splashpage. The actual URL for the repo would be design.canada.ca for the English content and conception.canada.ca for the French content. 

If "design" flies with OL/LO we might not need the pope's approval.

File: index.html (hunk 3 above)
Comment (delisma) [reply index.html:16]: > It seems like most of this should be coming from the root template, and this page just should be metadate only for the template values

That's why I did this file only. Wanted to see how the content should be added when there's a lot of customization to declare. Still figuring it out. Will update the Jekyll template.

File: index.html (hunk 1 above)
Comment (delisma) [reply index.html:2]: Wasn't sure either. Was just following WET-BOEW-documentation example...I think

File: index.html (hunk 3 above)
Comment (delisma) [reply index.html:16]: > We've also been adding a JSON-LD snippet for schema.org metadata, contextual to the page. How can we keep that when using jekyll?
> 
> See example on [radio buttons page](https://github.com/canada-ca/design-system/blob/master/components/radio-buttons.html)

I think it's doable with Jekyll. I'll check with @duboisp today

Comment (delisma) [reply index.html:16]: It's done in the wet-boew/gweb-jekyll repo that we're calling in through `remote_theme`

**Pull request closed without merging**